src/Service/MetaService.php

```diff
@@ -77,7 +77,7 @@ public function setEntity(?EntityMetaInterface $entity): void
         $this->entity = $entity;
     }
 
-    protected function getEntityMetaData(EntityMetaInterface $entity): array
+    public function getEntityMetaData(EntityMetaInterface $entity): array
     {
         return array_filter($entity->toMetaOGArray());
```

Comment:
@RobinHoutevelts The idea is to be able to override this method in our `wmcustom` `MetaService`, so the four-liner won't work.

Let's go for the breaking change then 👍

Reply:
@benjamintoussaint It does work


https://www.loom.com/share/ac79719fd751408a8fc23b0e11e6a9f8?sid=92287c32-2a99-4fbb-83ca-9f7a3aa12f6c

Reply:
@RobinHoutevelts I think that I got it 🌈 

I've overwritten the `getEntityMetadata` method in `Drupal\wmcustom\Service\MetaService`:

```php
public function getEntityMetaData(EntityMetaInterface $entity): array
{
    $defaultMetaData = parent::getDefaultMetaData();
    $metaData = parent::getEntityMetaData($entity);

    $this->metaTitleEditor->editMetaTitle($entity, $metaData, $defaultMetaData['site_name']);

    return $metaData;
}
```

And I was afraid that my `editMetaTitle` wouldn't be called anymore, but since `protected getEntityMetaData()` is called by `public getMetaData()`, I'm guessing that it could still work 👍 

It's way less clear about the intention, but in the short term, that's indeed less of a breaking change.

Reply:
@RobinHoutevelts I've deployed the updated `wmmeta` on watwat and it still works as expected, thanks for your help on this one, I think that we are done 👍 

❓ Should it be a `1.10.5` or rather a `1.11.0` (new feature)?

Reply:
@benjamintoussaint lets go for `1.11.0` then

Reply:
@RobinHoutevelts I've added `1.11.0` to the changelog.

Are we good now for merging and tagging?